src/main/java/core/basesyntax/JavaStreamApi.java

```diff
@@ -2,89 +2,77 @@
 
 import java.util.Collections;
 import java.util.List;
+import java.util.NoSuchElementException;
 import java.util.Optional;
+import java.util.stream.Collectors;
 
 public class JavaStreamApi {
 
-    /**
-     * <p>1. Дано: List of Integer numbers.
-     * Вернуть сумму нечетных числел или 0, если таких несуществует</p>
-     **/
     public Integer oddSum(List<Integer> numbers) {
-        return null;
+        return numbers.stream().filter(number -> number % 2 == 1).mapToInt(Integer::intValue).sum();
```

Comment:
https://mate-academy.github.io/jv-program-fulltime/01_core/checklist/hw_19.html

Reply:
try to use reduce

Reply:
done

Reply:
done